gtsfm/densify/mvs_math.py

```diff
@@ -0,0 +1,49 @@
+"""MVS math methods for gtsfm
+
+Authors: Ren Liu
+"""
+import math
+import numpy as np
+
+from gtsfm.utils.geometry_comparisons import angle_between_vectors
+
+
+def piecewise_gaussian(
+    xPa: np.ndarray, xPb: np.ndarray, theta_0: float = 5, sigma_1: float = 1, sigma_2: float = 10
+) -> float:
+    """Evaluate the similarity of vectors from a common track's coordinates to different cameras' centers in world frame
```

Comment:
What does a “common track’s coordinates” mean @codyly ?

Reply:
the coordinates for one track point in the two views

Reply:
I see. Currently the language makes me think that we are trying to merge two separate tracks.

Let's say instead
```
"Evaluate the angle between two rays, extending from the 3d coordinates of a track point to two different cameras (all in the world frame)
```